Core.ApplicationServices/Authorization/ResourcePermissionsResult.cs

```diff
@@ -35,7 +35,7 @@ public static Result<ResourcePermissionsResult, OperationError> FromEntity<T>(
             T entity,
             IAuthorizationContext authorizationContext) where T : IEntity
         {
-            return new ResourcePermissionsResult(true, authorizationContext.AllowModify(entity), authorizationContext.AllowDelete(entity));
+            return new ResourcePermissionsResult(authorizationContext.AllowReads(entity), authorizationContext.AllowModify(entity), authorizationContext.AllowDelete(entity));
```

Comment:
this change is not needed. If the code gets to this point it had already validated that the user has a read permission

some of the GetEntity method that proceed this call:
![image](https://github.com/user-attachments/assets/24889c41-278d-4c6f-8093-0545cfc163ea)
![image](https://github.com/user-attachments/assets/2fbc8aa1-ccc7-4fa6-9f91-58a97c412c5f)
![image](https://github.com/user-attachments/assets/1974c122-067c-4b50-9e8f-5989cb1419b9)

Reply:
I made that change because the updated (massive) service test started failing on the ones where read was false: 
![image](https://github.com/user-attachments/assets/51795bc9-2a27-4486-8ed2-59caf970ca64)
And since the function is public I prefer to have it covered in case it gets used somewhere without the prerequisite checks.